controller/cluster-controller.go

```diff
@@ -0,0 +1,44 @@
+package controller
+
+import (
+	"context"
+	"fmt"
+
+	"k8s.io/client-go/tools/cache"
+
+	apiscrv1    "github.com/grtl/mysql-operator/pkg/apis/cr/v1"
```

Comment:
Should be one space between the name and import, please gofmt your code.

Reply:
Fixed,